src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] agrs) {
+        Machine excavator = new Excavator();
```

Comment:
Let's create an array of machines, iterate over it and call required methods

Reply:
Done!